ManatiCR/manatime
# 7: Submit timesheet alert.

**Steps to test**
- Create some nodes: users, project, task, client, timesheet and time entries.
- Check if the button 'Submit Timesheet' run successfully.
- Check if the timesheet's submit form show an alert when you register less hours than the weekly hours.

[issue](https://github.com/ManatiCR/manatime/issues/7)
